LibreNMS/Data/Store/InfluxDBv2.php

```diff
@@ -62,6 +63,17 @@ public static function isEnabled()
      */
     public function put($device, $measurement, $tags, $fields)
     {
+        $device_data = Device::find($device['device_id']);
+        $device_groups = $device_data->groups;
```

Comment:
This is also an sql query, but if we pull the device from the device cache, hopefully it is only fetched once.  Might be worthwhile to check the sql queries during a poller run.

Reply:
Checked, it's only fetched once now that device_data is pulled from cache :-).